src/test/statistics/statistics_objects/generic_histogram_test.cpp

```diff
@@ -0,0 +1,902 @@
+#include <limits>
```

Comment:
does_not_contain was made public and is tested now. Are there other "things" that you cannot find tests for? 

Excuse my fingerpointing, but what is it with this newfound affection for doc'ing and testing internal functions? The TableScan internals, most of these `_scan_non_reference_segment`, `_simd_scan_with_iterators`, `_scan_generic_segment`, functions lack both doc and dedicated tests - and are iirc your creation.

Reply:
Documenting and testing internal functions are two different things. For the first, I find that it makes it easier to understand existing code. When reading the main method, it is helpful to have a quick summary of the helper methods that are called. Sure, for methods that span only a couple of lines, it's easy to read them, but still increases the complexity.

With testing, I don't think that every private method needs to be tested by itself. This one came up because I had a hard time finding which tests cover this.

It is hard to draw a precise line here. In cases where adding comments and/or tests would be a significant effort, I am more than willing to reconsider the PR comment. If existing code lacks comments and/or tests, feel free to open an issue for it. The fact that it's in the code base does not mean that it's perfect; no matter who wrote it.